constraintlayout/core/src/main/java/androidx/constraintlayout/core/state/ConstraintSetParser.java

```diff
@@ -970,11 +970,38 @@ private static void parseFlowType(String flowType,
 
                     }
                     break;
-                case "padding":
-                    String paddingValue = element.get(param).content();
+                case "paddingLeft":
```

Comment:
I think this instead should remain as "padding"
but the value may either be 1,2 or 4 values
1 value
```json5
 flow: {
    padding: 10 // applies to all values
 }
```
2 values
```json5
 flow: {
    padding: [5, 10] // 5 applies to horizontal values, 10 applies to vertical values
 }
```
4 values
```json5
 flow: {
    padding: [1, 2, 3, 4] // left, top, right, bottom
 }
```

See the API of Modifier.padding if you're not sure what I mean

Reply:
Thoughts @jafu888 ?

Reply:
Updated the code based on the suggestion!